server/re/src/mailMS.cpp

```diff
@@ -46,6 +56,24 @@
  * \sa none
  **/
 int msiSendMail( msParam_t* xtoAddr, msParam_t* xsubjectLine, msParam_t* xbody, ruleExecInfo_t* ) {
+    const auto enabled = [] {
+        try {
+            return irods::get_advanced_setting<bool>("enable_msiSendMail");
```

Comment:
what if it is set to 'apples'... will that set `enabled` to be true enough for line 71 below?

Reply:
Setting the value to apples will cause an exception to be thrown due to a type mismatch.

Resulting in `enabled` being set to false.